middlewares/tokenDecoded.js

```diff
@@ -0,0 +1,27 @@
+import jwtDecode from 'jwt-decode';
```

Comment:
What is the purpose of this file?
I believe we already have an auth middleware.

Reply:
earlier the team created the checkAuthentication middleware but when working on it, wrong data being spent skipped through, but we have adjusted the file  and I will revert  to using the one checkAuthentication middleware @Chidi3b3r3

Reply:
have deleted this file